multibranch/src/main/java/org/jenkinsci/plugins/workflow/multibranch/WorkflowMultiBranchProject.java

```diff
@@ -48,7 +48,7 @@
 @SuppressWarnings({"unchecked", "rawtypes"}) // core’s fault
 public class WorkflowMultiBranchProject extends MultiBranchProject<WorkflowJob,WorkflowRun> {
 
-    static final String SCRIPT = "jenkins.groovy";
+    static final String SCRIPT = "Jenkinsfile";
```

Comment:
> the file name should be configurable by the user

I agree, but that is a subject for a future feature (perhaps a form control in `WorkflowBranchProjectFactory`). This is about changing the default filename.